library/db/src/main/kotlin/org/keynote/godtools/android/db/ToolMapper.kt

```diff
@@ -55,5 +57,6 @@ internal object ToolMapper : BaseMapper<Tool>() {
         defaultOrder = c.getInt(COLUMN_DEFAULT_ORDER, 0)
         order = c.getInt(COLUMN_ORDER, Int.MAX_VALUE)
         isAdded = getBool(c, COLUMN_ADDED, false)
+        isHidden = getBool(c, COLUMN_HIDDEN, false)
```

Comment:
So I tracked this all the way down to https://github.com/CruGlobal/android-gto-support/blob/28f0bc272625df820e0cfe4a75c08e3e86969836/gto-support-db/src/main/java/org/ccci/gto/android/common/db/util/CursorUtils.java#L26 and just wanted to double-check something. I see in the ```tool_hidden_true.json``` you have ```"attr-hidden": "TruE"```. Does this get that value of ```TruE``` and then convert it to a boolean, or is there somewhere that does that and then passes it to this?

Reply:
The jaonapi converter takes care of converting Json to a boolean, which is handled in a different part of the app